### DISCUSSION TITLE
Marketing

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#1717] [Ideas]
Ex: Campaign Standard object (similar to Lists)

**Comment by AshHartwell** [comment]
Just a comment on pricing - your existing pricing methods ($29 per seat per month) are very similar to competitors, but the financials never work for our company (small business, 20 people). We all want access to see our pipeline, but licencing everyone would just be very expensive, so we then either have to settle for a (limited) free product or just licencing a few users (neither ideal).

Because of this, we are currently migrating from a free CRM account to... a spreadsheet.

Seems like pricing models just aren't designed for our kind of business. Would be great if there was pricing that was like x opportunities per month and x new contacts per month = $x.

Would be interested to see pricing that could work for this - maybe just a flat fee if below X deals. We are dealing in the hundreds of inbound enquiries per year category rather than the thousands.

**Reply by FelixMalfait** [reply]
Hey @AshHartwell great point. To be honest we haven’t figured out the pricing strategy at all and I agree with you that $29/ seat isn’t the right one.  Would you be open to chat? Would love to get your feedback on ideas we have that are more aligned with value actually created for end-users

https://cal.com/ifelix/30min > sorry I haven’t updated my availabilities there and I’m on my phone now. I’m busy from 2pm to 6:30pm CET every week these days but available for every other slots (morning CET or last slot at night CET). Feel free to book directly!

**Reply by kesowkk** [reply]
Hi @FelixMalfait If you are looking to choose a right pricing strategy, let me help you to find the one. Please let me know.

**Reply by FelixMalfait** [reply]
https://cal.com/ifelix/25min always happy to chat about that feel free to book a call or reach out on Discord 
https://discord.gg/cx5n4Jzs57 thanks!
